services/src/main/java/org/keycloak/authentication/forms/RegistrationRecaptcha.java

```diff
@@ -130,4 +130,5 @@ public List<ProviderConfigProperty> getConfigProperties() {
         properties.addAll(super.getConfigProperties());
         return properties;
     }
+
```

Comment:
Just a nitpick. This seems like an unintentional change caused by the rebase.

Reply:
Indeed, removed.